gigamon/logs/Gigamon-test.yml

```diff
@@ -0,0 +1,30 @@
+id: gigamon
+tests:
+  - sample: '{"dns_query":"daisy.ubuntu.com","generator":"gs_apps_appInst73_4231e20c-bbb7-56c7-3464-66f73ee65fd1","dns_class":"1","src_packets":"0","seq_num":"4276353","dst_ip":"8.8.8.8","device_inbound_interface":"0","src_ip":"10.60.80.152","intf_name":"0","protocol":"6","ip_version":"4","dns_opcode":"0","vendor":"Gigamon","src_bytes":"0","dns_qdcount":"1","dns_transaction_id":"37003","dns_host_type":"A","id":"4982150692664770561","app_id":"32","dst_bytes":"274","src_mac":"00:1b:17:00:37:31","sys_up_time_first":"61264","dns_flags":"33152","end_reason":"1","dns_host_addr":"162.213.35.25","dns_ancount":"2","end_time":"2024:05:03
```

Comment:
These saples don't seem properly formatted since the result is `null`. Can you fix the unnecessary new line characters that are breaking the sample?